packages/frontend/app/(dashboard)/course/[cid]/components/QueueCard.tsx

```diff
@@ -80,7 +83,15 @@ const QueueCard: React.FC<QueueCardProps> = ({
           <span className="mr-8 flex flex-row flex-wrap items-center justify-between">
             <div>
               {queue.room}
-              <div className="flex">
+              <div className="mb-1 flex flex-wrap gap-y-1 sm:mb-0">
+                {queue?.type && (
+                  <Tag
+                    color="#2a9187"
+                    className="m-0 mr-1 leading-4 text-gray-200"
+                  >
```

Comment:
![image](https://github.com/user-attachments/assets/eb0d8c1e-4095-4d4b-9865-41c0b8be26ea)
I noticed the tags' colors are the same for each location. I believe I may have briefly mentioned when you were starting this issue to try having a different color for each location (like green for in-person, teal for hybrid, and blue for online). I don't blame you if you forgot though, or I may be misremembering completely anyway.

Reply:
![image](https://github.com/user-attachments/assets/343b5c7d-e829-487c-b7f7-5ca1b7c029c9)

What do you think about these colors?

Reply:
i tried to keep them fitting the theme but realized its really hard to distinguish them (which would be the point of this) unless we have some warm colors in there too.

Reply:
Ah I see yeah fair enough. I like your color choice for these, though I think maybe it stands out a little too much maybe? Hybrid tag looks good, but I wonder how it would look like if we chose some shades of color that are closer to the color of the Queue Card header for In-Person and Online?

![image](https://github.com/user-attachments/assets/cd35fed5-a415-4619-b86c-07b8bd54e163)
![image](https://github.com/user-attachments/assets/045ecaa3-ccaf-4175-91a2-2279939c35e9)
Or in other words, maybe try what it looks like if we choose colors that don't contrast as much